### PR TITLE
LPS-144410

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-144410

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss [hunk 1]
```diff
@@ -59,7 +59,6 @@
 	.autosave-feedback-container {
 		align-items: center;
 		display: flex;
-		justify-content: center;
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:62]: @rodrigopaulino can you please check if the behavior described in [LPS-120480](https://issues.liferay.com/browse/LPS-120480) is happening after your changes?

**Comment by carolmariaabb** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:62]: Hi, @carolmariaabb.

This is what happens:
Screen Shot 2022-01-04 at 10 26 34

Not the same behavior, less critical than what's been reported in the past imo, but not ideal either.
I'd say to drop this change from the PR (the message will move around when changing tabs, but that's the current behavior) and open a technical debt to deal with the responsiveness of the navigation bar altogether. Right now, we have too many buttons and no responsive solution for this.

Here's Forms in current 7.4 in an iPhone X simulation:
Screen Shot 2022-01-04 at 10 40 44

iPad simulation:
Screen Shot 2022-01-04 at 10 41 01

What do you think?

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:62]: >  open a technical debt to deal with the responsiveness of the navigation bar altogether.

I totally agree with you!

Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss]: Great. It's done. I'll open a technical debt for that. Thanks, @carolmariaabb

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Here's the technical debt [ticket](https://issues.liferay.com/browse/LPS-144792).

**Review by carolmariaabb — APPROVED** [review]

**Pull request closed without merging**